### PR TITLE
Ft17 18 21

### PR DESCRIPTION
**Pull request by ricky1uphealth**
#17 #18 #21

### CODE REVIEW

File: spec/export/index.md [hunk 1]
```diff
@@ -96,21 +96,38 @@ Export data from a FHIR server, whether or not it is associated with a patient.
...
+    <th>Optionality</th>
+    <th>Type</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>```_outputFormat```</td>
+      <td>Optional</td>
+      <td>String</td>
+      <td>  The format for the requested bulk data files to be generated as per [FHIR Asynchronous Request Pattern](http://hl7.org/fhir/async.html). Defaults to `application/fhir+ndjson`. Servers SHALL support [Newline Delimited JSON](http://ndjson.org), but MAY choose to support additional output formats. Servers SHALL accept the full content type of ```application/fhir+ndjson``` as well as the abbreviated representations ```application/ndjson``` and ```ndjson```.</td>
```
Comment (jmandel) [inline spec/export/index.md]: @ricky1uphealth The `-based formatting doesn't work inside HTML tables (or at least, isn't rendering for me). I think we need `<code>` formatting for these.

File: spec/export/index.md [hunk 2]
```diff
@@ -299,36 +316,36 @@ Content-Type: application/json
 Required Fields:
```
Comment (jmandel) [inline spec/export/index.md]: The aim from https://github.com/HL7/bulk-data/issues/21 is that these JSON fields will be documented in their own table, as https://cds-hooks.org/specification/1.0/

image

File: spec/index.md [hunk 3]
```diff
@@ -34,3 +34,10 @@ This use case exports all resources needed to convey a patient's healththcare fi
 * [Client and Server Implementations](https://github.com/smart-on-fhir/fhir-bulk-data-docs/blob/master/implementations.md)
 * [Discussion Group (FHIR Zulip "Bulk Data" Track)](https://chat.fhir.org/#narrow/stream/bulk.20data)
 * [Argonaut Project: Bulk Data Export Security Risk Assessment Report](authorization/security-risk-assessment-report.pdf)
+
+## Additional Documentation
+* [Overview](index.html)
```
Comment (jmandel) [inline spec/index.md]: Don't need a self-link here.e

Comment (ricky1uphealth) [reply spec/index.md]: yup agree, https://github.com/HL7/bulk-data/pull/34/commits/b6732c25263e611d2bab1a25763d8d86b9fe089a

File: spec/export/index.md (hunk 1 above)
Comment (ricky1uphealth) [reply spec/export/index.md]: got it, we have to use code tags for it work well, fixed here https://github.com/HL7/bulk-data/pull/34/commits/b090adf1b0a7427d0ddcb7eda99275d238142859

File: spec/export/index.md (hunk 2 above)
Comment (ricky1uphealth) [reply spec/export/index.md]: @jmandel gotcha, `result body` i was thinking req params.  https://github.com/HL7/bulk-data/pull/34/commits/fd7f9d4c7d76a542fa570913ccc3b68a2fae8881

**Comment by ricky1uphealth** [comment]
@jmandel i think this is good to go now.

File: spec/export/index.md [hunk 4]
```diff
@@ -298,38 +313,91 @@ Content-Type: application/json
...
+<table class="table">
+  <thead>
+    <th>Parameters</th>
+    <th>Optionality</th>
+    <th>Type</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>alg</code></td>
```
Comment (jmandel) [inline spec/export/index.md]: What is this? Is this `<tr>` a copy/paste error?

File: spec/export/index.md [hunk 5]
```diff
@@ -96,21 +96,36 @@ Export data from a FHIR server, whether or not it is associated with a patient.
...
-
-  Resource references MAY be relative URLs with the format `<resource type>/<id>`, or absolute URLs with the same structure rooted in the base URL for the server from which the export was performed. References will be resolved by looking for a resource with the specified type and id within the file set.
-
-  For example  `_type=Practitioner` could be used to bulk data extract all Practitioner resources from a FHIR endpoint.
+<table class="table">
+  <thead>
+    <th>Parameters</th>
```
Comment (jmandel) [inline spec/export/index.md]: This should say "Parameter" or "Query Parameter"

File: spec/export/index.md [hunk 6]
```diff
@@ -298,38 +313,91 @@ Content-Type: application/json
...
-  - ```error``` - array of error file items following the same structure as the `output` array.
-
-    Errors that occurred during the export should only be included here (not in output). If no errors occurred, the server SHOULD return an empty array.  Only the `OperationOutcome` resource type is currently supported, so a server SHALL generate files in the same format as bulk data output files that contain `OperationOutcome` resources.
-
-The response body and any file item MAY optionally contain the following field:
-
-   - ```extension``` - To support extensions, this implementation guide reserves the name ```extension``` and will never define a field with that name, allowing server implementations to use it to provide custom behavior and information. For example, a server may choose to provide a custom extension that contains a decryption key for encrypted ndjson files. The value of an extension element SHALL be a pre-coordinated JSON object.
+<table class="table">
+  <thead>
+    <th>Parameters</th>
```
Comment (jmandel) [inline spec/export/index.md]: This should say "JSON Property" or "Field"

**Comment by jmandel** [comment]
Thanks @ricky1uphealth  -- shared a few more comments.

File: spec/export/index.md (hunk 4 above)
Comment (ricky1uphealth) [reply spec/export/index.md]: yup thanks for catching that. fixed here https://github.com/HL7/bulk-data/pull/34/commits/810c7fa9f769f0fb3b16162dbc6e343062141a83

File: spec/export/index.md (hunk 5 above)
Comment (ricky1uphealth) [reply spec/export/index.md]: https://github.com/HL7/bulk-data/pull/34/commits/4951c8b82051e242541a9066b4959da450978bbf

File: spec/export/index.md (hunk 6 above)
Comment (ricky1uphealth) [reply spec/export/index.md]: fixed https://github.com/HL7/bulk-data/pull/34/commits/3b992e89ec2a9dbbf54004d9d3b0691846e66865

**Comment by ricky1uphealth** [comment]
@jmandel hey josh i addressed all your comments yesterday. Is there anything else on this?

**Pull request merged**